docs/modules/reference/pages/configuration/performance-data-collection/introduction.adoc

```diff
@@ -0,0 +1,6 @@
+[[ref-performance-data-collectors]]
+= Performance data collectors
```

Comment:
I would keep the name "Collectors" for this section to match what it was previously and to match how they are referred within the collectd config file.
```suggestion
= Collectors
```